plugins/services/src/js/components/modals/ServiceModals.js

```diff
@@ -7,6 +7,7 @@ import ServiceTree from '../../structs/ServiceTree';
 import ServiceActionItem from '../../constants/ServiceActionItem';
 import ServiceDestroyModal from './ServiceDestroyModal';
 import NewCreateServiceModal from './NewCreateServiceModal';
+import NewApplicationDefaults from '../../constants/NewApplicationDefaults';
```

Comment:
I actually like your approach, because it makes it a bit more clear that this is a constant, and it also solves the naming issues. Perhaps we should run it through the team sync.